src/pages/SubmissionDetails.js

```diff
@@ -36,34 +39,80 @@ export const SubmissionDetails = ({ submissionId }) => {
       }
     }
 
+    // TODO: "Prework" for the next ticket
+    // const loadSubmissionData = async () => {
+    //   try {
+    //     const runSetData = await Ajax(signal).Cbas.runSets.get()
+    //     setRunSetData(runSetData.run_sets)
+    //     setSubmissionTimestamp(JSON.parse(runSetData.submission_timestamp))
+    //   } catch (error) {
+    //     notify('error', 'Error loading run set data', { detail: await (error instanceof Response ? error.text() : error) })
+    //   }
+    // }
+
     loadRunsData()
+    //loadSubmissionData()
   })
 
+
   const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, runsData)
+  // TODO: "Prework" for the next ticket
+  //const specifyRunSet = _.filter(r => r.run_set_id === submissionId, runSetData)
 
   const firstPageIndex = (pageNumber - 1) * itemsPerPage
   const lastPageIndex = firstPageIndex + itemsPerPage
   const paginatedPreviousRuns = sortedPreviousRuns.slice(firstPageIndex, lastPageIndex)
 
   const rowWidth = 100
-  const rowHeight = 200
+  const rowHeight = 50
   return h(Fragment, [
-    headerBar(),
-    div({ style: { margin: '4em' } }, [
-      div({ style: { display: 'flex', marginTop: '0.5rem', justifyContent: 'space-between' } }, [
+    div({
+      style: {
+        borderBottom: '2px solid rgb(116, 174, 67)',
+        boxShadow: 'rgb(0 0 0 / 26%) 0px 2px 5px 0px, rgb(0 0 0 / 16%) 0px 2px 10px 0px',
+        position: 'relative'
+      }
+    }, [
+      headerBar(),
+      div({ style: { marginLeft: '4em', display: 'flex', marginTop: '0.5rem', justifyContent: 'space-between' } }, [
         h1(['Submission Details']),
         h(ButtonOutline, {
+          style: { margin: '2em' },
           onClick: () => Nav.goToPath('root')
         }, ['Submit a new workflow'])
       ]),
-      div([
-        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`])
-      ]),
+      div({ style: { marginLeft: '4em' } }, [
+        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`]),
+        h2(['workflow_name(HARDCODED)']),
+        h3([`Submission date: `]),
+        h3(['Duration: ']),
+        h3(['Submitted by: '])
+      ])
+    ]),
+    div({
+      style: {
+        backgroundColor: 'rgb(235, 236, 238)',
+        display: 'flex',
+        flex: '1 1 auto',
+        flexDirection: 'column',
+        padding: '1rem 3rem'
+      }
+    }, [
       div({
         style: {
           marginTop: '1em', height: tableHeight({ actualRows: paginatedPreviousRuns.length, maxRows: 12.5, heightPerRow: 250 }), minHeight: '10em'
         }
       }, [
+        div([h2(['Workflows'])]),
+        div([h3(['Filter by: '])]),
+        h(Select, {
+          isDisabled: false,
+          'aria-label': 'Select a data table',
```

Comment:
is this label correct? I could imagine be able to filter workflows by data table, but not for it to be the only way to filter